task_sdk/src/airflow/sdk/api/datamodels/_generated.py

```diff
@@ -126,9 +126,12 @@ class TerminalTIState(str, Enum):
     """
 
     SUCCESS = "success"
-    FAILED = "failed"
+    FAILED = "failed"  # This state indicates that we attempt to retry.
     SKIPPED = "skipped"
     REMOVED = "removed"
+    FAIL_WITHOUT_RETRY = (
+        "fail_without_retry"  # This state is useful for when we want to terminate a task, without retrying.
+    )
```

Comment:
Auto-generation will nuke these comments.

```suggestion
    FAILED = "failed"
    SKIPPED = "skipped"
    REMOVED = "removed"
    FAIL_WITHOUT_RETRY = "fail_without_retry"
```

Reply:
Right, removed it